R/req-perform-stream.R

```diff
@@ -129,10 +130,44 @@ req_perform_connection <- function(req, blocking = TRUE) {
   resp
 }
 
-# TODO: max_size
+#' @export
+#' @rdname req_perform_connection
+close.httr2_response <- function(con, ...) {
+  check_streaming_response(con)
```

Comment:
Does this mean if you call `close(resp)` on a non-streaming response, you'll get an error? If so, I think a no-op would be better; as a general rule I try not to throw in cleanup code since it's so often invoked in places where errors are not expected or hard to deal with (like in a `finally` or a gc finalizer).

Reply:
Yeah makes sense.